enzyme/test/Integration/ForwardModeVector/binops.cpp

```diff
@@ -0,0 +1,42 @@
+// RUN: %clang++ -std=c++11 -fno-exceptions -O0 %s -S -emit-llvm -o - | %opt - %loadEnzyme -enzyme -S | %lli -
```

Comment:
Can you make this a C test rather than C++?

Reply:
fixed